umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md

```diff
@@ -92,41 +100,34 @@ For any other microservice, follow these steps to enable the LoadBalancer servic
    ```
 
    Replace `<external-ip>` with the external IP address of the node.
-4. Scroll to the `spec.type` section and change the value from ClusterIP to
-   LoadBalancer.
-5. Click **Save** to apply the changes.
+4. Go to the `spec.type` section and change the value from `ClusterIP` to
+   `LoadBalancer`. 
+5. After saving, your changes will be applied automatically and the service will 
+   be updated. Now, you can access the service with the configured address.
```

Comment:
"Now, you can access the service at the configured address."

Reply:
updated